docs/usage/executor/x.md

```diff
@@ -0,0 +1,84 @@
+---
+id: x
+title: "X plugin"
```

Comment:
As discussed in previous PR, please consider changing a name to a bit more verbose one 👍